source/introduction/software-overview.rst

```diff
@@ -51,7 +51,7 @@ In some cases, though, when an existing application must not be altered (e.g., w
 Using :code:`AppRun` is not a guarantee that an application will run, and the packager must provide all the resources an application could need manually (or by using external tools) before creating the AppImage with :ref:`appimagetool`. :code:`AppRun` force-changes the current working directory, and therefore applications can not detect where the AppImage was called originally. This may be especially annoying for CLI tools, but can also be a problem for GUI applications expecting paths via parameters.
 
 .. note::
-   :code:`AppRun` is legacy technology, and should be avoided if possible. Tools like :ref:`linuxdeploy` deploy applications in a different way, and deprecated its usage. This doesn't mean there's no cases in which :code:`AppRun` might be useful, but it's got several limitations a user must be aware of before using it.
+   An :code:`AppRun` binary or script may not be needed in some cases, and a symlink with the same name to the main binary may be sufficient instead. Tools like :ref:`linuxdeploy` deploy applications in a different way.
```

Comment:
IMO it should still be stated that ***this*** AppRun is legacy technology and it definitely should be avoided. It does forced `cwd` changes etc., which is not modern and annoying and causes issues.

If any, you should change the wording in a way that the binary AppRun released in AppImageKit should be avoided.

The AppRun symlink of course is still necessary. I think something like the following might work:

```rst
The binary :code:`AppRun` released as part of AppImageKit is legacy technology and should be avoided if possible. It requires some nasty and troublesome mechanisms (like e.g., changing the current working directory), which can cause hard to debug issues for AppImage authors.

Using compiled binaries or scripts for :code:`AppRun` is, in most cases, obsolete, and modern tools like :ref::`linuxdeploy` deploy applications in a way so that a simple symlink to the main binary that is called :code:`AppRun` will be sufficient.